workflow-dev/manifests/deis-logger-fluentd-daemon.yaml

```diff
@@ -20,7 +21,7 @@ spec:
       serviceAccount: deis-logger-fluentd
       containers:
       - name: deis-logger-fluentd
-        image: quay.io/deisci/fluentd:canary
+        image: quay.io/deisci/fluentd:208d0dc
         imagePullPolicy: Always
         securityContext:
           privileged: true
```

Comment:
can you remove this file and ensure that its templated version is in `workflow-dev/tpl`?